server/db_models/README.md

```diff
@@ -0,0 +1,21 @@
+## Database setup
+
+1. The postgreSQL server is running, and the database you want to use is ready.
```

Comment:
It might be worth mentioning the version of Postgres that should be used.

Reply:
agreed, the app is developed with postgreSQL 12.3; no guarantees with other postgres versions.